gateway/src/main/resources/application.yml

```
@@ -54,3 +52,6 @@ eureka:
     register-with-eureka: true
     serviceUrl:
       defaultZone: ${EUREKA_SERVER_URL:http://localhost:8070/eureka/}
+
+jwt:
+  secret: ${JWT_SECRET}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)